src/query/service/src/interpreters/interpreter_copy_v2.rs

```diff
@@ -42,7 +42,7 @@ use crate::sql::plans::Plan;
 use crate::storages::stage::StageTable;
 
 const MAX_QUERY_COPIED_FILES_NUM: usize = 50;
-
+const TABLE_COPIED_FILE_KEY_EXPIRE_AT: Option<u64> = Some(7);
```

Comment:
Is there a list of all the magic numbers we are using? How do we prevent the use of duplicated numbers here?

Reply:
This magic number limit in the file only, does not exported outside as public.

Reply:
> This magic number limit in the file only, does not exported outside as public.

This is why it's easier to be broken. We could declare the duplicated numbers.

Reply:
For now, this magic number is used in this file only, and there is no file to save these const vars.